--- origsrc/gnome-weather-3.8.2/libgd/Makefile.am	2013-05-04 07:57:08.000000000 -0500
+++ src/gnome-weather-3.8.2/libgd/Makefile.am	2013-08-06 09:13:59.882107300 -0500
@@ -22,7 +22,7 @@ pkglib_LTLIBRARIES = libgd.la
 endif
 
 libgd_la_LIBADD = $(LIBGD_LIBS) $(LIBM)
-libgd_la_LDFLAGS = -avoid-version
+libgd_la_LDFLAGS = -module -avoid-version -no-undefined
 libgd_la_SOURCES = libgd/gd.h
 nodist_libgd_la_SOURCES =
 
